examples/beam/pom.xml

```diff
@@ -31,17 +31,11 @@ under the License.
     <name>Nemo Examples: Beam</name>
 
     <dependencies>
-        <dependency>
-            <groupId>org.apache.nemo</groupId>
-            <artifactId>nemo-compiler-frontend-beam</artifactId>
-            <version>${project.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.nemo</groupId>
-            <artifactId>nemo-client</artifactId>
-            <version>${project.version}</version>
-            <scope>test</scope>
-        </dependency>
+      <dependency>
+        <groupId>org.apache.nemo</groupId>
+        <artifactId>nemo-client</artifactId>
+        <version>${project.version}</version>
+      </dependency>
```

Comment:
Please revert this change

Reply:
I think it actually makes sense that the examples package depends on the client. I've found out that without this, it causes some problem while running on YARN environments. I'll keep you updated with what I'm investigating.